administrator/language/en-GB/plg_multifactorauth_email.sys.ini

```
@@ -0,0 +1,7 @@
+; Joomla! Project
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_EMAIL="Multi-factor Authentication - Authentication Code by Email"
+PLG_MULTIFACTORAUTH_EMAIL_XML_DESCRIPTION="Use time-limited, six-digit security codes sent to you by email."
```

```suggestion
PLG_MULTIFACTORAUTH_EMAIL_XML_DESCRIPTION="Use time limited, six digit security codes sent to you by email."
```

its an americanisation to try and hyphenate everything.